packages/web3-eth/test/fixtures/rpc_methods_wrappers.ts

```diff
@@ -1461,7 +1461,7 @@ const receiptInfoNumberString: FormatType<
 	blockNumber: '6139707',
 	cumulativeGasUsed: '13244',
 	gasUsed: '1244',
-	status: '1',
+	status: '0x1',
```

Comment:
Shouldn't it be formatted to `1`?

Reply:
It should be `1`. I created an issue to to look into it later. https://github.com/ChainSafe/web3.js/issues/4980